### PR TITLE
Added the parametric `rayTriangle` intersection test to the API as `rayTriangleParametric`

### PR DESCRIPTION
**Pull request by likangning93**
See https://github.com/AnalyticalGraphicsInc/gltf-pipeline/pull/102.

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -7,6 +7,9 @@
           <option name="SPACE_BEFORE_PROPERTY_COLON" value="true" />
           <option name="SPACE_BEFORE_FUNCTION_LEFT_PARENTH" value="false" />
         </JSCodeStyleSettings>
+        <MarkdownNavigatorCodeStyleSettings>
```
Comment (pjcozzi) [inline .idea/codeStyleSettings.xml]: Did you mean to submit this?

File: Source/Core/IntersectionTests.js [hunk 2]
```diff
@@ -78,7 +78,19 @@ define([
...
+     *
+     * @param {Ray} ray The ray.
+     * @param {Cartesian3} p0 The first vertex of the triangle.
+     * @param {Cartesian3} p1 The second vertex of the triangle.
+     * @param {Cartesian3} p2 The third vertex of the triangle.
+     * @param {Boolean} [cullBackFaces=false] If <code>true</code>, will only compute an intersection with the front face of the triangle
+     *                  and return undefined for intersections with the back face.
+     * @returns {Number} The intersection as a parametric distance along the ray, or undefined if there is no intersection.
+     */
+    IntersectionTests.rayTriangleParametric  = function(ray, p0, p1, p2, cullBackFaces) {
```
Comment (pjcozzi) [inline Source/Core/IntersectionTests.js]: Does this need new unit tests?  Or is it fully covered by the tests for `IntersectionTests.rayTriangle`?

**Comment by pjcozzi** [comment]
Update CHANGES.md so users know about this.

**Comment by likangning93** [comment]
Updated! This should be completely covered by the tests for `IntersectionTests.rayTriangle`. The most significant functional difference between the two is that `rayTriangleParametric` can return intersections behind the ray origin (negative parametric distance) but that functionality it seems to be covered by the `does not intersect behind the ray origin` test.

**Comment by pjcozzi** [comment]
Looks great, thanks @likangning93!

**Pull request merged**